platform/backend/repository/testbed/src/test/kotlin/io/hamal/repository/CodeTest.kt

```diff
@@ -450,29 +452,84 @@ internal class CodeRepositoryTest : AbstractUnitTest() {
 
     }
 
-    private fun CodeRepository.createCode(
-        codeId: CodeId,
-        groupId: GroupId,
-        cmdId: CmdId = CmdId(abs(Random(10).nextInt()) + 10),
-        codeValue: CodeValue
-    ) {
-        create(
-            CreateCmd(
-                id = cmdId,
-                codeId = codeId,
-                groupId = groupId,
-                value = codeValue
+    @Nested
+    inner class TimestampTest {
+
+        @TestFactory
+        fun `Initialization Test`() = runWith(CodeRepository::class) {
+            createCode(CodeId(1), GroupId(1), CmdGen(), CodeValue("1+1"))
+            Thread.sleep(1000)
```

Comment:
This is pretty bad - 1 second is an eternity. At the moment I dont have a better idea for this myself - lets keep this, but not add more tests like this one in the future